js_modules/dagster-ui/packages/ui-core/src/assets/AssetSidebarActivitySummary.tsx

```diff
@@ -60,6 +62,20 @@ export const AssetSidebarActivitySummary = ({
         </>
       )}
 
+      {pools.length ? (
+        <SidebarSection title={pools.length === 1 ? 'Pool' : 'Pools'}>
+          <Box margin={{horizontal: 24, vertical: 12}} flex={{gap: 12, alignItems: 'flex-start'}}>
+            <Body style={{flex: 1}}>
+              <Box flex={{gap: 4}}>
+                {pools.map((pool, idx) => (
+                  <PoolTag key={idx} pool={pool} />
+                ))}
+              </Box>
+            </Body>
+          </Box>
```

Comment:
- Are all of the flex properties necessary? Can it just be one `Box` that contains the `PoolTag` items?
- Is the `Body` wrapper necessary?